### PR TITLE
fix forking console

### PR DESCRIPTION
**Pull request by raubitsj**
Fixes WB-11880

Description
-----------
What does the PR do?

Testing
-------
How was this PR tested?

Checklist
-------
- [ ] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#conventional-commits)

### CODE REVIEW

File: wandb/sdk/wandb_run.py [hunk 1]
```diff
@@ -1377,6 +1377,10 @@ def _console_callback(self, name: str, data: str) -> None:
 
     def _console_raw_callback(self, name: str, data: str) -> None:
         # logger.info("console callback: %s, %s", name, data)
+        console_pid = getattr(self, "_attach_pid", 0)
```
Comment (dmitryduev) [inline wandb/sdk/wandb_run.py:1380]: nice!

**Comment by kptkin** [comment]
@raubitsj oh interesting, so our decorator doesn't work here? do you know why?

**Comment by raubitsj** [comment]
> @raubitsj oh interesting, so our decorator doesn't work here? do you know why?

yep.. the decorator was only for top level methods for the run object, this was a callback that we installed on the write method of stdout/stderr...   we might be able to do an attach from there, but it just scares me so the safe thing for now is to ignore stdout/stderr output from forked processes.

**Comment by raubitsj** [comment]
Closed in favor of consolidated https://github.com/wandb/wandb/pull/4877 which was just merged

**Pull request closed without merging**